### ISSUE TITLE
301 redirect from carbondesignsystem.com to www.carbondesignsystem.com

### ISSUE THREAD

**Issue by shdq** [carbon-design-system/carbon-website#758]
Hello, I hope you are doing well.

There is no redirection from carbondesignsystem.com to www.carbondesignsystem.com.
And maybe you should consider making it in the opposite way from www to no-www because it's an archaism. Anyway, some users can't reach the site.

Thank you!

**Comment by alisonjoseph** [comment]
This should be fixed now!
